styles/style.css

```diff
@@ -0,0 +1,16 @@
+
+
+
+
+header{
+
+    background-color: var(--green);
+    color: var(--grey);
+    font-family: var(--primary-font);
+    font-size: 12.5rem;
+    height: 15.625rem;
```

Comment:
The height you can leave it as px. We can work more tomorrow to know when and why use one or another